pkg/api/types.go

```diff
@@ -204,7 +204,7 @@ type VolumeSource struct {
 	// Glusterfs represents a Glusterfs mount on the host that shares a pod's lifetime
 	Glusterfs *GlusterfsVolumeSource `json:"glusterfs"`
 	// PersistentVolumeClaimVolumeSource represents a reference to a PersistentVolumeClaim in the same namespace
-	PersistentVolumeClaimVolumeSource *PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim,omitempty"`
+	PersistentVolumeClaimVolumeSource *PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim"`
```

Comment:
omitempty implies optional. Really we shouldn't have any json tags in this file, but so long as we do, we might as well make the tags correct and consistent. All volume sources should have omitempty here, or just remove the json tags entirely.